core/misc/dropbutton.css

```diff
@@ -5,6 +5,14 @@
  */
 
 /**
+ * Set width of dropbuttons where extra width is needed
+ */
+#system-modules .dropbutton-wrapper div,
+#layout-edit .dropbutton-multiple div,
+#layout-list .dropbutton-wrapper div {
+  min-width: 140px;
+}
```

Comment:
Now split into https://github.com/backdrop/backdrop/pull/853 and https://github.com/backdrop/backdrop/pull/854